RecoBTag/TensorFlow/test/plotDDX.py

```diff
@@ -0,0 +1,74 @@
+#    edm::ValueMap<float>                  "offlineSlimmedPrimaryVertices"   ""               "PAT"       
+#    vector<pat::Jet>                      "slimmedJetsAK8"            ""               "PAT"       
+#    vector<pat::Jet>                      "slimmedJetsAK8PFPuppiSoftDropPacked"   "SubJets"        "PAT"       
+#    vector<reco::Vertex>                  "offlineSlimmedPrimaryVertices"   ""               "PAT"       
+#    vector<reco::VertexCompositePtrCandidate>    "slimmedSecondaryVertices"   ""               "PAT"       
+#    edm::OwnVector<reco::BaseTagInfo,edm::ClonePolicy<reco::BaseTagInfo> >    "selectedUpdatedPatJets"    "tagInfos"       "PATtest"   
+#    edm::OwnVector<reco::BaseTagInfo,edm::ClonePolicy<reco::BaseTagInfo> >    "updatedPatJets"            "tagInfos"       "PATtest"   
+#    edm::OwnVector<reco::BaseTagInfo,edm::ClonePolicy<reco::BaseTagInfo> >    "updatedPatJetsTransientCorrected"   "tagInfos"       "PATtest"   
+#    vector<CaloTower>                     "selectedUpdatedPatJets"    "caloTowers"     "PATtest"   
+#    vector<pat::Jet>                      "selectedUpdatedPatJets"    ""               "PATtest"   
+#    vector<pat::Jet>                      "updatedPatJets"            ""               "PATtest"   
+#    vector<pat::Jet>                      "updatedPatJetsTransientCorrected"   ""               "PATtest"   
+#    vector<reco::BoostedDoubleSVTagInfo>    "pfBoostedDoubleSVAK8TagInfos"   ""               "PATtest"   
+#    vector<reco::FeaturesTagInfo<btagbtvdeep::DeepDoubleBFeatures> >    "pfDeepDoubleBTagInfos"     ""               "PATtest"   
+#    vector<reco::GenJet>                  "selectedUpdatedPatJets"    "genJets"        "PATtest"   
+#    vector<reco::PFCandidate>             "selectedUpdatedPatJets"    "pfCandidates"   "PATtest"   
+
+h_probQ_ddb = ROOT.TH1F('h_probQ_ddb', ';prob Q;', 40, 0., 1.)
+h_probH_ddb = ROOT.TH1F('h_probH_ddb', ';prob H;', 40, 0., 1.)
+h_probQplusH_ddb = ROOT.TH1F('h_probQplusH_ddb', ';prob Q + H;', 44, 0., 1.1)
+
+h_probQ_ddc = ROOT.TH1F('h_probQ_ddc', ';prob Q;', 40, 0., 1.)
+h_probH_ddc = ROOT.TH1F('h_probH_ddc', ';prob H;', 40, 0., 1.)
+h_probQplusH_ddc = ROOT.TH1F('h_probQplusH_ddc', ';prob Q + H;', 44, 0., 1.1)
+
+for iev,event in enumerate(events_c):
+    event.getByLabel (labelJ, handleJ)
+    jets = handleJ.product()
+    for jet in jets  :
+	if jet.pt() < 300 or jet.pt() > 2000: continue
+	if jet.mass() < 40 or jet.mass() > 200: continue
+
+        print jet.pt(), jet.mass()
+        print "DDB", jet.bDiscriminator("pfDeepDoubleBvLJetTags:probQCD"), jet.bDiscriminator("pfDeepDoubleBvLJetTags:probHbb") , jet.bDiscriminator("pfDeepDoubleBvLJetTags:probHcc")
+        print "DDB", jet.bDiscriminator("pfMassIndependentDeepDoubleBvLJetTags:probQCD"), jet.bDiscriminator("pfMassIndependentDeepDoubleBvLJetTags:probHbb") , jet.bDiscriminator("pfMassIndependentDeepDoubleBvLJetTags:probHcc")
```

Comment:
\*BvLJetTags:probHcc doesn't look like a legal option.
Does this script work?

Reply:
It's not, it was a cross check for what happens if one tries to access that - > - 1000

Reply:
it would be nice to cleanup this script from clearly broken parts.

Reply:
@andrzejnovak this issue was not solved if I am not mistaken